assets/package.json

```diff
@@ -11,7 +11,8 @@
   },
   "dependencies": {
     "phoenix": "file:../deps/phoenix",
-    "phoenix_html": "file:../deps/phoenix_html"
+    "phoenix_html": "file:../deps/phoenix_html",
+    "sass": "^1.80.3"
```

Comment:
question: is this right? there's a runtime dependency on `sass`?

Reply:
Seems so, I had to remove `node-sass` and replace it with this otherwise the assets wouldn't load because `node-sass` had some sort of python issue

Reply:
And it's needed at runtime (i.e. not in `devDependencies`)?